src/transferslot.cpp

```diff
@@ -1083,7 +1097,21 @@ void TransferSlot::doio(MegaClient* client)
         }
     }
 
-    p += transfer->progresscompleted;
+    if (transfer->type == GET && transferbuf.isRaid())
+    {
+        p = transferbuf.progress();
```

Comment:
Any way we can rename `p` to `currentProgress` or something?

Reply:
That variable has been there such a long time and I it's usage is quite compilcated and spread through this large function.  I think it would need a thorough analysis to ensure the name matched its use properly.   In this case I'm just changing its value only for the isRaid() case, before it's existing use in reporting progress.